provider.go

```diff
@@ -151,7 +151,7 @@ func (d DiffResponse) rpc() *rpc.DiffResponse {
 		DeleteBeforeReplace: d.DeleteBeforeReplace,
 		Changes:             rpc.DiffResponse_DIFF_NONE,
 		DetailedDiff:        detailedDiff(d.DetailedDiff).rpc(),
-		HasDetailedDiff:     true,
+		HasDetailedDiff:     len(d.DetailedDiff) == 0,
```

Comment:
`> 0`?

Reply:
🤦 Thanks for the catch. This will need a test before merging.